lms/djangoapps/teams/static/teams/js/spec_helpers/team_discussion_helpers.js

```diff
@@ -0,0 +1,126 @@
+define([
+    'underscore', 'common/js/spec_helpers/ajax_helpers'
+], function (_, AjaxHelpers) {
+    'use strict';
+    var createMockPostResponse, createMockDiscussionResponse, createAnnotatedContentInfo, createMockThreadResponse,
+        testCourseID = 'course/1',
+        testUser = 'testUser',
+        testTeamDiscussionID = "12345";
+
+    createMockPostResponse = function(options) {
+        return _.extend(_.extend({
```

Comment:
This is an idiom I've used before, in that we want a new dict that is then pre-populated with defaults, and then overridden with the provided options. I think I had issues when extending the default dict directly that it ended up corrupting it for subsequent calls. However, I'm not sure about that now. Also, I messed the idiom up here because it doesn't provide the initial empty dict.

Let me try it without the extra level and see if it works...